.github/workflows/verify-jdk9-classes.sh

```diff
@@ -17,12 +17,10 @@
 # specific language governing permissions and limitations
 # under the License.
 
-echo "Checking Scala CLI version..."
-scala-cli --version
-
 echo "Starting verification with Scala-CLI..."
 
 for file in stream/target/scala-cli/*.sc; do
+  cat $file
```

Comment:
is this a debugging leftover? should it remain in?

Reply:
@raboof I think it is useful to have the generated script in the logs but if you think it best, I can remove this line,

Reply:
if it's intentional it's fine by me